draft-ZIP-0226.rst

```diff
@@ -0,0 +1,303 @@
+
+As was initially proposed by Jack Grigg and Daira Hopwood [#initial-zsa-issue]_, we propose to make this happen by changing the value base point, :math:`\mathcal{V}^{\mathsf{Orchard}}`, in the Homomorphic Pedersen Commitment that generates the value commitment, :math:`\mathsf{cv^{net}}`, of the *net value* in an Orchard Action.
+
+Because in a single transaction all value commitments are balanced, there must be as many different value base points as there are asset types in the transaction. We propose to make the :math:`\mathsf{type}` identifier an auxiliary input to the proof, represented already as a point in the Pallas curve. The circuit then should check that the same :math:`\mathsf{type}` is used in the old note commitment, in the new note commitment **and** in the value commitment as the base point of the value. This ensures that (1) the input and output notes are of the same type, and (2) that only actions with the same asset type identifier will balance out in the binding and balance signature.
+
+In order to ensure the security of the transfers, and as we will explain below, we are replacing input dummy notes for custom assets, as we need to enforce that the type of the output note of that split action is the output of a PRF.
+
+Finally, in this ZIP we also describe the *burn* mechanism, which is a direct extension of the transfer mechanism. The burn process uses a similar mechanism than what is used in Orchard to unshield ZEC, by using the :math:`\mathsf{valueBalance}` of the asset in question. Burning assets is useful for many purposes, including bridging of wrapped assets and removing supply of native assets.
+
+Orchard Protocol Changes
+========================
+
+Most of the protocol is kept the same as the Orchard protocol released with NU5, except the following that will be deployed to mainnet in subsequent network upgrades.
+
+Asset Types
+-----------
+
+In the OSA protocol, we include a new variable, the asset type identifier, :math:`\mathsf{type}`, which is generated as a 32-byte string during issuance (as described in the Issuance ZIP [#zip-0227]_). The :math:`\mathsf{type}` will then be publicly hashed into the corresponding group, in this case the Pallas curve, by using the :math:`\mathsf{GroupHash}`
+function. In fact, every ZSA note will contain the group element representation of the asset type identifier. This will enable a much more elegant and simple version of the circuit, as we will see.
```

Comment:
```suggestion
function. In fact, every ZSA note will contain the group element representation of the Asset type identifier. This will enable a more elegant and simple version of the circuit, as we will see.
```

More elegant and simple relative to?

Reply:
This portion has been rewritten in [PR#649](https://github.com/zcash/zips/pull/649), we have removed the "more elegant and simple" statement.